### PR TITLE
ansible playbooks for prod and local dev vagrant deployment

### PR DESCRIPTION
**Pull request by andrewschoen**

### CODE REVIEW

File: deploy/playbooks/roles/common/templates/prod.py.j2 [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        """),
+        'keyname': 'jenkins-build',
+        'image_name': 'Ubuntu Trusty (2014.04) cloudimg',
+        'size': 'c3.mxlarge',
+        'labels': ['amd64', 'x86_64', 'precise', 'trusty', 'wheezy', 'jessie'],
+        'provider': 'openstack'
+    },
+    'wheezy': {
+        'script': dedent("""#!/bin/bash
+        curl -L "http://172.18.181.11/setup/slave/?token=727999862a610c7436318580de6f3e27&executors=1&labels=amd64+precise+trusty+wheezy+jessie+x86_64&nodename=wheezy__%s" | bash
```
Comment (alfredodeza) [inline deploy/playbooks/roles/common/templates/prod.py.j2]: tokens should be private (not pushed here) so maybe a variable is needed here as well

File: deploy/playbooks/roles/common/templates/prod_provider.py.j2 [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+provider = {
```
Comment (alfredodeza) [inline deploy/playbooks/roles/common/templates/prod_provider.py.j2]: they depend. e.g. these things will certainly be different with, say, DreamHost

**Comment by andrewschoen** [comment]
@alfredodeza I moved the nodes configuration into it's own file and created a ``prado_url`` and ``jenkins_prado_token`` to configure that section.

File: deploy/playbooks/roles/common/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+- include: circus.yml
+  tags:
+    - circus
+
+- include: nginx.yml
+  tags:
+    - nginx
+
+# nginx should be up and running by now, there is a bug with Ubuntu 14.04 and
```
Comment (alfredodeza) [inline deploy/playbooks/roles/common/tasks/main.yml]: I don't think we need mita exposed via nginx

**Pull request merged**